packages/core/src/types.ts

```diff
@@ -2348,19 +2348,21 @@ export type ToChildren<TActor extends ProvidedActor> =
       // or maybe even `TActor["logic"]` since it's possible to configure `{ src: string; logic: SomeConcreteLogic }`
       // TODO: consider adding `| undefined` here
       Record<string, AnyActorRef>
-    : ToConcreteChildren<TActor> &
-        {
-          include: {
-            [id: string]: TActor extends any
-              ? ActorRefFrom<TActor['logic']> | undefined
-              : never;
-          };
-          exclude: {};
-        }[undefined extends TActor['id'] // if not all actors have literal string IDs then we need to create an index signature containing all possible actor types
-          ? 'include'
-          : string extends TActor['id']
+    : Compute<
```

Comment:
This just brings back `Compute` here so it's essentially a revert of this change https://github.com/statelyai/xstate/pull/4706/files#r1467593357